extensions/spark/kyuubi-spark-authz/src/test/resources/sparkSql_hive_jenkins.json

```diff
@@ -463,6 +463,64 @@
       "guid": "fd24db19-f7cc-4e13-a8ba-bbd5a07a2d8d",
       "isEnabled": true,
       "version": 5
+    },
+    {
+      "service": "hive_jenkins",
+      "name": "src_key _less_than_20",
+      "policyType": 2,
+      "policyPriority": 0,
+      "description": "",
+      "isAuditEnabled": true,
+      "resources": {
+        "database": {
+          "values": [
+            "default"
+          ],
+          "isExcludes": false,
+          "isRecursive": false
+        },
+        "table": {
+          "values": [
+            "src"
+          ],
+          "isExcludes": false,
+          "isRecursive": false
+        }
+      },
+      "policyItems": [],
+      "denyPolicyItems": [],
+      "allowExceptions": [],
+      "denyExceptions": [],
+      "dataMaskPolicyItems": [],
+      "rowFilterPolicyItems": [
+        {
+          "rowFilterInfo": {
+            "filterExpr": "key\u003c20"
```

Comment:
Can `\u003c` be replaced by `<`?

Reply:
don't know, the item is generated by ranger admin